.github/workflows/ci.yml

```
@@ -1,59 +1,59 @@
-    name: Spelling
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout branch
-        uses: actions/checkout@v4
-
-      - name: Setup
-        uses: ./.github/actions/setup
-
-      - name: Run spelling check
-        run: bun check:spelling
+name: CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)